src/setup/joystick.c

```diff
@@ -1155,7 +1155,7 @@ static void AdjustAnalog(TXT_UNCAST_ARG(widget), TXT_UNCAST_ARG(unused))
         TXT_NewLabel("Turn"),
         TXT_NewSpinControl(&joystick_x_sensitivity, 0, 20),
         NULL);
-    if (gamemission == heretic || gamemission == hexen || gamemission == strife)
```

Comment:
Why is Strife removed here? I thought it had looking up and down. If so, it should be possible to use gamepad for that.

Reply:
@haleyjd Opinions on Strife gamepad up/down look?

Reply:
Now that I've made it so 0 sensitivity disables inputs in non-analog mode, I've restored the look sensitivity option in chocolate-strife-setup.